system-addon/content-src/asrouter/components/SnippetBase/_SnippetBase.scss

```diff
@@ -3,10 +3,11 @@
   bottom: 0;
   left: 0;
   right: 0;
-  background-color: var(--newtab-card-background-color);
+  background-color: var(--newtab-snippets-background-color);
   color: var(--newtab-text-primary-color);
   font-size: 12px;
   line-height: 16px;
+  border-top: solid 1px var(--newtab-snippets-hairline-color);
   box-shadow: $inner-box-shadow;
```

Comment:
This is actually supposed to be `$shadow-secondary`, do you mind fixing that while you're here?

Reply:
for sure! done